### DISCUSSION TITLE
Calendar set year first

### DISCUSSION THREAD

**Discussion by pabloaleman** [primefaces/community#2027] [PrimeVue]
There is any way to set the calendar that by default the user can select first the year, then the month and finally the day?
First
<img width="275" alt="Captura de pantalla 2024-05-26 a la(s) 10 56 47" src="https://github.com/primefaces/community/assets/19631755/6c910889-2f10-49b3-891d-ae516c624b67">

then the month
<img width="274" alt="Captura de pantalla 2024-05-26 a la(s) 10 56 57" src="https://github.com/primefaces/community/assets/19631755/370c4b79-d61b-4670-8211-6fc7b3b3875d">

Finally the day
<img width="284" alt="Captura de pantalla 2024-05-26 a la(s) 10 57 05" src="https://github.com/primefaces/community/assets/19631755/3f6aeaff-7e0b-4c21-af49-a99a68b030c7">

the result
<img width="225" alt="Captura de pantalla 2024-05-26 a la(s) 10 57 13" src="https://github.com/primefaces/community/assets/19631755/704d87f4-bc7c-49a8-a864-66132d949af3">

**Comment by MacaRamos** [comment]
I need the same, were you able to do it?

**Reply by pabloaleman** [reply]
Hi
I do a component with 3 calendar that show: only year, only month and only day.
I attach the component.
I hope it helps you
[CalendarAnioMes.txt](https://github.com/user-attachments/files/20613582/CalendarAnioMes.txt)
[CalendarAnioMesDia.txt](https://github.com/user-attachments/files/20613583/CalendarAnioMesDia.txt)

**Reply by MacaRamos** [reply]
Muchas gracias ☺️
